src/applications/static-pages/alerts-dismiss-view.js

```diff
@@ -21,43 +21,33 @@ export function alertsBuildShow() {
     content.setAttribute('aria-hidden', ariaHidden);
   }
 
-  const elements = document.querySelectorAll('.usa-alert-text');
-  let i;
-  for (i = 0; i < elements.length; i++) {
-    if (
-      elements[i].id &&
-      document.querySelectorAll(
-        `#${elements[i].id} .field--name-field-text-expander`,
-      ).length
-    ) {
-      const clicker = document.querySelector(
-        `#${elements[i].id} .field--name-field-text-expander`,
-      );
+  Array.from(document.querySelectorAll('.usa-alert-text'))
```

Comment:
There's a bug in this file around dismissing alerts when you're on landing pages, so I fixed that and cleaned up some of the loops in here.